front/src/components/Ants.vue

```diff
@@ -38,12 +44,19 @@ export default {
   },
   methods: {
     buildAntConfig (ant) {
-      return {
+      const antConfig = {
```

Comment:
Whichever way we do these, let's be consistent, or it will be very confusing for me

Reply:
I think we will definitely need a Mapper contruct in the frontend ... unless the mapper construct on the backend does the coordinate translation for us?
I'll try and follow my boss Wes' Rule Of Three: when I repeat something for the third time I'll abstract it.